src/modules/proposal/services/proposal-misc.service.ts

```diff
@@ -454,8 +462,35 @@ export class ProposalMiscService {
       throw new ForbiddenException('Only FDPG members can update participants after draft/FDPG_CHECK status');
     }
 
+    const oldParticipants = [...proposal.participants];
     mergeDeep(proposal, { participants });
 
+    // Compare old participants with the actual merged state
+    addHistoryItemForParticipantsUpdated(proposal, user, oldParticipants, proposal.participants);
+
+    const savedProposal = await proposal.save();
+    return plainToClass(ProposalGetDto, savedProposal.toObject(), {
+      strategy: 'excludeAll',
+      groups: [ProposalValidation.IsOutput],
+    });
+  }
+  async removeParticipant(id: string, participantId: string, user: IRequestUser): Promise<ProposalGetDto> {
+    const proposal = await this.proposalCrudService.findDocument(id, user, undefined, true);
+
+    if (!this.canUpdateParticipants(proposal, user)) {
+      throw new ForbiddenException('Only FDPG members can remove participants after draft/FDPG_CHECK status');
+    }
+
+    const participantIndex = proposal.participants.findIndex((p) => p._id.toString() === participantId);
+    if (participantIndex === -1) {
+      throw new NotFoundException('Participant not found');
```

Comment:
I wonder if we should really throw here. In my opinion, the desired status (participant not on the proposal) is there. If for any reasons there was a misalignment between frontend and backend before, it might be in sync after we ignore this case gracefully.

Of course, we wouldn't write any history item, or trigger side effects. Just let the frontend "think" everything was fine